contracts/README.md

```diff
@@ -1,8 +1,28 @@
 # Nori Smart Contracts
 
+Nori's product is the NRT or Nori _Removal_ Tonne which can be thought of
+as a carbon credit granted to a supplier for the CO2 they have removed from
+the atmosphere.
+The supplier consigns this asset in Nori's marketplace and gets paid in _NORI_ token.
+A buyer of NRTs is issued a _Certificate_
+(NCCR) which is non-transferrable and owns the NRTs that were swapped from
```

Comment:
Could be a little clearer here - does the Certificate own the NRTs, or does the buyer own the NRTs? If the buyer owns the NRTs, can they resell them?